app/Renderers/Layouts/Calendar/Reminder.php

```diff
@@ -27,7 +27,7 @@ public function columns(): array
                 'key' => 'type_id',
                 'label' => 'crud.fields.entity_type',
                 'render' => function ($model) {
-                    return $model->remindable->entityType->name();
+                    return $model->remindable?->entityType?->name() ?? __('entities.post') . ' (' . $model->remindable->entity->entityType->name() . ')';
```

Comment:
Wrong logic, should be `$model->remindable instanceof Post ? __('entities.post') : else`